### PR TITLE
Added Blog image and information

### PR DESCRIPTION
**Pull request by DotSlashTX**
Blog title: root to Open-Source with FOSSASIA
img: gutter.jpg

### CODE REVIEW

**Review by akshatnitd — CHANGES_REQUESTED** [review]
You have added the image twice. Remove the unneccesary one and also try to squash the commits

**Comment by akshatnitd** [comment]
@FreakinBabaJi Great work! Now squash the commits, so that I can go ahead and merge the PR.

File: _data/blogs.yml [hunk 1]
```diff
@@ -42,3 +42,8 @@
   title: Interview with Prasenjit Gautam, GCI Contestant and Open Source Contributor
   img: 7.jpg
   author_name: Kuba Michalski  
+  
+- url: https://codegutter.wordpress.com/2017/12/09/root-to-open-source-with-fossasia/
+  title: Root to Open-Source with FOSSASIA
+  img: gutter.jpg
```
Comment (CloudyPadmal) [inline _data/blogs.yml]: Change this to `rsz_gutter.jpg` as it is the file name :)

**Review by DotSlashTX — COMMENTED** [review]
named the image as it was in img/blog

**Comment by akshatnitd** [comment]
@FreakinBabaJi Please resolve the conflicts and squash the commits properly

**Comment by DotSlashTX** [comment]
resolved all conflicts :)

**Review by ParthS007 — APPROVED** [review]
LGTM ! Good work , If you can squash commits it would be a plus 👍

**Comment by ParthS007** [comment]
@FreakinBabaJi , You can go through this https://gist.github.com/patik/b8a9dc5cd356f9f6f980 for squash ing of commits.

**Comment by ParthS007** [comment]
@FreakinBabaJi  I am merging this PR ,Try to squash commits from next PR 👍

**Pull request merged**